osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -169,25 +172,32 @@ private double computeSpeedValue()
 
         private double computeAccuracyValue()
         {
-            // This percentage only considers HitCircles of any value - in this part of the calculation we focus on hitting the timing hit window
-            double betterAccuracyPercentage;
             int amountHitObjectsWithAccuracy = Attributes.HitCircleCount;
 
-            if (amountHitObjectsWithAccuracy > 0)
-                betterAccuracyPercentage = ((countGreat - (totalHits - amountHitObjectsWithAccuracy)) * 6 + countOk * 2 + countMeh) / (double)(amountHitObjectsWithAccuracy * 6);
-            else
-                betterAccuracyPercentage = 0;
+            if (amountHitObjectsWithAccuracy == 0)
+                return 0;
 
-            // It is possible to reach a negative accuracy with this formula. Cap it at zero - zero points
-            if (betterAccuracyPercentage < 0)
-                betterAccuracyPercentage = 0;
+            // This section should be documented by Tr3, but effectively we're calculating the exact same way as before, but
+            // we calculate a variance based on the object count and # of 50s, 100s, etc. This prevents us from having cases
+            // where an SS on lower OD is actually worth more than a 95% on OD11, even though the OD11 requires a greater
+            // window of precision.
 
-            // Lots of arbitrary values from testing.
-            // Considering to use derivation from perfect accuracy in a probabilistic manner - assume normal distribution
-            double accuracyValue = Math.Pow(1.52163, Attributes.OverallDifficulty) * Math.Pow(betterAccuracyPercentage, 24) * 2.83;
+            double p100 = (2 * (double)countOk) / amountHitObjectsWithAccuracy; // this is multiplied by two to encourage better accuracy. (scales better)
+            double p50 = (1 * (double)countMeh) / amountHitObjectsWithAccuracy;
+            double pm = (1 * (double)countMiss) / amountHitObjectsWithAccuracy;
```

Comment:
Don't these miss/meh/ok counts also include these types results from sliders, not only normal circles? This seems important given that `amountHitObjectsWithAccuracy` is equal to hitcircle count, so this can artificially inflate scores that miss a lot of circles but hit a lot of sliders in proportion? I'm not really sure how huge of a deal this is but I'll note it down for posterity.

Reply:
No real way around this, the buff for these maps is lower anyway since the count of hitobjects with accuracy is typically lower. seems negligable.